crates/release_util/src/bin/release-util.rs

```diff
@@ -1,3 +1,63 @@
-fn main() {
-    println!("Hello, release-util!");
+use anyhow::Context;
+use clap::{Parser, Subcommand};
+use release_util::{generate_release, publish_release};
+use std::path::PathBuf;
+
+#[derive(Parser)]
+pub struct ReleaseUtilCli {
+    /// The directory to run the command in.
+    ///
+    /// Defaults to the current directory.
+    #[arg(long, default_value = ".")]
+    dir: PathBuf,
+
+    #[command(subcommand)]
+    command: ReleaseUtilCommand,
+}
+
+#[derive(Subcommand)]
+pub enum ReleaseUtilCommand {
+    /// Generate changes for the next release.
+    Generate {
```

Comment:
Or something like that, which describes what is being generated?
```suggestion
    PrepareNext {
```

Reply:
Would love to keep this a single word but I agree the one I'd picked wasn't clear - switched to `Prepare`, does that work?

https://github.com/holochain/release-integration/pull/3/commits/a7090a20f07bf2cc1be9d650f1cbb9a5e7146c0c